src/Accounts/Accounts/AzureRmAlias/Mappings.json

```diff
@@ -1632,6 +1632,7 @@
     "New-AzFirewallNatRule": "New-AzureRmFirewallNatRule",
     "New-AzFirewallNetworkRuleCollection": "New-AzureRmFirewallNetworkRuleCollection",
     "New-AzFirewallNetworkRule": "New-AzureRmFirewallNetworkRule",
+    "New-AzFirewallThreatIntelWhitelist": "New-AzureRmFirewallThreatIntelWhitelist",
```

Comment:
You don't need to modify `Mappings.json`, new cmdlets shouldn't support `AzureRm`

Reply:
changed